gdb-xtensa-esp32-elf/meta.yaml

```diff
@@ -1,9 +1,9 @@
 {% set name = "xtensa-esp32-elf-gdb" %}
-{% set version = "11.1_20220318" %}
+{% set version = "12.1_20221002" %}
 
 package:
   name: {{ name }}
-  version: {{ version }}.memfault1
+  version: {{ version }}.memfault0
```

Comment:
Wow! I thought we *were* on 12 for ESP32? Nice!

Reply:
hah me too!